OpenRA.Mods.Common/Scripting/Global/TriggerGlobal.cs

```diff
@@ -407,14 +407,14 @@ public void OnInfiltrated(Actor a, LuaFunction func)
 
 		[Desc("Call a function when this actor is discovered by an enemy (doesn't work for neutral actors). " +
 			"The callback function will be called as func(Actor self, Player discoverer)")]
```

Comment:
No, I didn't. The callback function *will* have that argument.  Again, the callback is *not* the function below this line. The callback is actually the `func` argument, which will be stored in the `ScriptTriggers` trait and then called when the event happens (on line 335 of `ScriptTriggers.cs`).